File: src/main/java/com/nicholasnassar/sfsapi/data/gpa/GPAClass.java
package com.nicholasnassar.sfsapi.data.gpa;

import io.vertx.core.json.JsonObject;

public class GPAClass {
    private final String name, letterGrade, points;

    public GPAClass(String name, String letterGrade, String points) {
        this.name = name;

        this.letterGrade = letterGrade;

        this.points = points;
    }

    public JsonObject asJson() {
        return new JsonObject().put("name", name).put("letter_grade", letterGrade).put("points", points);
    }
}

File: src/main/java/com/nicholasnassar/sfsapi/data/gpa/GPAClass.kt
package com.nicholasnassar.sfsapi.data.gpa

import io.vertx.core.json.JsonObject

class GPAClass(val name: String, val letterGrade: String, val points: String) {
    fun asJson(): JsonObject {
        return JsonObject().put("name", name).put("letter_grade", letterGrade).put("points", points)
    }
}
